doc/examples/transform/plot_steerable.py

```diff
@@ -0,0 +1,96 @@
+"""
+=================
+Steerable Pyramid
+=================
+
+The steerable pyramid is a multi-resolution image decomposition
+with directional selectivity [1]_.
+
+The function `steerable.build_steerable` takes a gray scale image
+as input, and returns a list of lists of subbands.
+
+The first element of the list is a numpy array represents highpass.
+The last element of the list is a numpy array represents lowpass.
+Intermediate elements are lists of subband at the same radius level
+but different orientations.
+
+
+.. [1] Simoncelli, E.P. & Freeman, W.T.
+       (1995). The Steerable Pyramid: A Flexible Architecture for Multi-Scale
+       Derivative Computation. In Proc. 2nd IEEE International Conf. on Image
+       Proc., vol.III pp. 444-447, Oct 1995. 
+       http://www.cns.nyu.edu/~eero/steerpyr/,
+       http://www.cns.nyu.edu/pub/eero/simoncelli95b.pdf,
+       DOI:10.1109/ICIP.1995.537667
```

Comment:
Could you add a short explanation of the use of these pyramids, other than plotting the coefficients?